src/main/java/org/opensearch/ad/constant/ADCommonName.java

```diff
@@ -72,4 +75,10 @@ public class ADCommonName {
     public static final String DUMMY_AD_RESULT_ID = "dummy_ad_result_id";
     public static final String DUMMY_DETECTOR_ID = "dummy_detector_id";
     public static final String CUSTOM_RESULT_INDEX_PREFIX = "opensearch-ad-plugin-result-";
+
+    // ======================================
+    // Insights job
+    // ======================================
+    // The Insights job name
+    public static final String INSIGHTS_JOB_NAME = "insights_job";
```

Comment:
how about changing to ad_insights_job in case we need forecasting job later?

Reply:
updated in the new revision